seatunnel-e2e/seatunnel-connector-v2-e2e/connector-file-local-e2e/src/test/resources/json/fake_to_local_file_json.conf

```diff
@@ -77,5 +77,7 @@ sink {
     file_format_type = "json"
     filename_time_format = "yyyy.MM.dd"
     is_enable_transaction = true
+    "schema_save_mode"="RECREATE_SCHEMA"
+    "data_save_mode"="DROP_DATA"
```

Comment:
Please check whether savemode is effective

Reply:
added